### PR TITLE
Add documentation for running development environment in kind

### PR DESCRIPTION
**Pull request by TheRealHaoLiu**
Signed-off-by: Hao Liu <haoli@redhat.com>

##### SUMMARY
Add instruction for AWX development on MacOS using Kind Cluster

##### ISSUE TYPE
 - New or Enhanced Feature

##### COMPONENT NAME
 - Docs

##### AWX VERSION
```
awx: 21.4.1.dev12+g7c8e12bc91
```

##### ADDITIONAL INFORMATION

```

```

### CODE REVIEW

File: docs/development/kind.md [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+NAME                                              READY   STATUS    RESTARTS   AGE
+awx-56fbfbb6c8-jkhzl                              4/4     Running   0          13h
+awx-operator-controller-manager-b775bfc7c-fn995   2/2     Running   0          16h
+awx-postgres-0                                    1/1     Running   0          16h
+
+# now you know the pod name, tail logs from task container in pod
+kubectl logs -f awx-56fbfbb6c8-jkhzl -n awx  -c awx-task
+
+# alternatively, or in a different window tail logs from the web container in pod
+kubectl logs -f awx-56fbfbb6c8-jkhzl -n awx  -c awx-web
```
Comment (shanemcd) [inline docs/development/kind.md]:
```
```suggestion
kubectl logs -f deployment/awx -n awx -c awx-web
```

File: docs/development/kind.md [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+```
+
+To exec in to the container:
+```
+# Using same pod name we found above from "kubectl get pods"
+$ kubectl exec -it awx-56fbfbb6c8-k6p82 -c awx-task bash
```
Comment (shanemcd) [inline docs/development/kind.md]:
```suggestion
$ kubectl exec -it deployment/awx -c awx-task bash
```

**Comment by TheRealHaoLiu** [comment]
updated instruction with suggestion from @shanemcd 
added note about using `stern` to tail log if there's multiple replica of the deployment

**Comment by john-westcott-iv** [comment]
Is there a provision to start up the dev UI as well?

**Review by shanemcd — APPROVED** [review]

**Pull request merged**